File: c++ programming/arrayOddeven.cpp
#include<iostream>
using namespace std;
int main()
{
	int i,arr[5];
	for(i=0;i<5;i++)
	{
		cout<<"Enter number : ";
		cin>>arr[i];
	}
	for(i=0;i<5;i++)
	{
		if(arr[i]%2==0)
		{
			cout<<"Even num is : "<<arr[i]<<endl;
		}
		else
		{
			cout<<"Odd num is : "<<arr[i]<<endl;
		}
	}
	return 0;
}

File: C#/Array_Odd_Even.cs
using System;


namespace Array_Seperate_Odd_Even
{
    internal class Program
    {
        static void Main(string[] args)
        {
            int[] arr1 = new int[5];
            int[] arr2 = new int[5];
            int[] arr3 = new int[5];
            for (int i = 0; i < 5; i++)
            {
                Console.WriteLine("Enter Num: ");
                arr1[i] = Convert.ToInt32(Console.ReadLine());
            }
            int j = 0, k = 0;
            for (int i = 0; i < 5; i++)
            {
                if (arr1[i] % 2 == 0)
                {
                    arr2[j] = arr1[i];
                    j++;
                }
                else
                {
                    arr3[k] = arr1[i];
                    k++;
                }
            }
            Console.WriteLine("Even no: ");
            for (int i = 0; i < j; i++)
            {
                Console.WriteLine("{0} ", arr2[i]);
            }
            Console.WriteLine("Odd no: ");
            for (int i = 0; i < k; i++)
            {
                Console.Write("{0} ", arr3[i]);
            }
            Console.ReadKey();

        }
    }
}
